### PR TITLE
Fixes bike dismanting

### PR DESCRIPTION
**Pull request by hal9000PR**
## What Does This PR Do
you can no longer dismantle the arcade bicycle

## Why It's Good For The Game
likely an oversight, you probably shouldn't be able to destroy 10k tickets worth of work with a single click of a wrench.

## Changelog
:cl:
Fix: you can no longer dismantle the arcade bicycle
/:cl:

### CODE REVIEW

**Comment by Herder05** [comment]
pain

**Comment by S34NW** [comment]
ah yes, that well known structure type `stool_bed_chair_nest`
Good fix

**Comment by DogsParadise** [comment]
I imagined you had messed up this pr

File: code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm [hunk 1]
```diff
@@ -156,3 +156,6 @@
 
 		else
 			. = 1
+
+/obj/structure/chair/wheelchair/bike/wrench_act(mob/user, obj/item/I)
```
Comment (AffectedArc07) [inline code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm:160]: `/obj/structure/chair/wheelchair/bike`

What

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**